insights/parsers/gcp_license_codes.py

```diff
@@ -0,0 +1,60 @@
+"""
+GCPLicenseCodes
+===============
```

Comment:
@SteveHNH please include the `.rst` file in the directory `insights/docs/share_parsers_catalog/` so that this new parser will be included in the documentation build.

Reply:
Ah! i keep forgetting that part. on it